src/index.html

```diff
@@ -1,13 +1,295 @@
+              </p>
+            </article>
+
+            <article class="detail">
+              <div class="detail__photos">
+                <img
+                  src="./images/detalis-img/3--1.png"
+                  alt="Auto Unlock"
+                  class="detail__photo detail__photo--square"
+                >
+
+                <img
+                  src="./images/detalis-img/3--2.png"
+                  alt="Auto Unlock"
+                  class="detail__photo detail__photo--wide"
+                >
+              </div>
+
+              <h3 class="detail__title">
+                Hydraulic disc brakes & Lightweight
+              </h3>
+
+              <p class="detail__text">
+                Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+              </p>
+            </article>
+
+            <article class="detail">
+              <div class="grid--position">
+                <button type="button" class="detail__button">
+                  Explore
+                </button>
+              </div>
+            </article>
+          </div>
+        </section>
+
+        <section class="contacts" id="contacts">
+          <h2 class="contacts__title">
+            Contact us
+          </h2>
+
+          <div class="grid--position">
+
+            <div class="contacts__content">
+
+              <input
+                type="name"
+                class="contacts__input contacts__input--name"
+                placeholder="Name"
+              >
+
+              <input
+                type="email"
+                class="contacts__input contacts__input--email"
+                placeholder="Email"
+              >
+
+              <textarea
+              class="contacts__message"
+              placeholder="Massage"
+            ></textarea>
+
+              <button type="button" class="contacts__button">
+                Send
+              </button>
+            </div>
+
+            <div class="contacts__info">
+              <p class="contacts__info__title">
+                Phone
+              </p>
+              <p class="contacts__info__text">
+                +1 234 5555-55-55
+              </p>
+              <p class="contacts__info__title">
+                Email
+              </p>
+              <p class="contacts__info__text">
+                hello@miami.com
+              </p>
+              <p class="contacts__info__title">
+                Address
+              </p>
+              <p class="contacts__text">
+                400 first ave.<br>suite 700<br>Minneapolis, MN 55401
+              </p>
+            </div>
+>
+          </div>
+        </section>
+      </main>
+    </div>
+
+    <footer class="footer">
+      <img
+        src="../src/images/footer.png.png"
+        alt="footer"
+        class="footer__img"
+      >
+    </footer>
```

Comment:
Add image as a background for the footer
```suggestion
    <footer class="footer">
    </footer>
```